test/Y2ModuleConfig_test.rb

```diff
@@ -22,7 +43,7 @@
         [
           "audit-laf", "autofs", "ca_mgm", "firstboot", "language", "restore",
           "runlevel", "sshd", "sysconfig", "unknown_profile_item_1",
-          "unknown_profile_item_2", "users"
```

Comment:
This has a purpose, although users_auto might be supported in general, they are not supported if they are not installed in the system - that's the trick